### DISCUSSION TITLE
Function to pull team description within Get-VenafiTeam?

### DISCUSSION THREAD

**Discussion by JamesM-NC** [Venafi/venafips#116] [Q&A] [answered]
I am curious if there is a way to include the team description while doing a pull using Get-VenafiTeam? If not, are there plans to add it in the future? Where would I put that type of request... I'm new to GitHub and this community.  :)  Thank you in advance for the help!

**Comment by gdbarron** [comment, marked as answer]
Hi @JamesM-NC, welcome!  You can submit a Feature Request by going to Issues->New Issue and selecting the appropriate type of issue, in this case Feature Request, and filling in all the appropriate fields.
